lib/ThreeScaleClient.php

```diff
@@ -38,14 +40,15 @@ class ThreeScaleClient {
    * @param $httpClient Object Object for handling HTTP requests. Default is CURL. Don't change it
    *                    unless you know what you are doing.
    */
-  public function __construct($providerKey = null, $host = self::DEFAULT_HOST, $port = self::DEFAULT_PORT, $httpClient = null) {
+  public function __construct($providerKey = null, $host = self::DEFAULT_HOST, $port = self::DEFAULT_PORT, $scheme = self::DEFAULT_SCHEME, $httpClient = null) {
```

Comment:
What I meant was not adding another param for scheme but unifying all those 3 into a url like `https://su1.3scale.net`. There is no need for 3 separate params when 1 url serves the same purpose.